app/templates/profile.html

```diff
@@ -0,0 +1,33 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <div class="text-center">
+        {% with messages = get_flashed_messages() %}
```

Comment:
We don't need flash here, there are in base template

Reply:
ok, you're right